tools/releasestaging

```diff
@@ -24,6 +24,33 @@ function osc_getbinaries
     return $ret
 }
 
+function check_installability
+{
+    repo=$1
```

Comment:
in OBS terms, this is a "project" or short "prj" - you can have multiple repositories in a Project as build-targets. E.g. "standard" is a repo here.